src/js/shield_defs.js

```diff
@@ -551,6 +565,9 @@ export function loadShields(shieldImages) {
   // Yukon
   shields["CA:YT"] = roundedRectShield(Color.shields.white, Color.shields.red);
 
+  // Costa Rica
+  shields["CR:national"] = badgeWithStrokeShield;
```

Comment:
`network=CR:national` is being tagged on every route in the [National Road Network](https://es.wikipedia.org/wiki/Anexo:Red_Vial_Nacional_de_Costa_Rica), making no distinctions between primary, secondary, and tertiary routes, even though all three classes have different shields. This is similar to the situation in https://github.com/ZeLonewolf/openstreetmap-americana/issues/455#issuecomment-1166417586.

`network=CR:national` está aplicado en cada ruta de la [Red Vial Nacional](https://es.wikipedia.org/wiki/Anexo:Red_Vial_Nacional_de_Costa_Rica), sin distinción entre las rutas primarias, secundarias, y tercerarias, aunque las tres clasificaciones tienen los señales diferentes. Es similar con la situación en https://github.com/ZeLonewolf/openstreetmap-americana/issues/455#issuecomment-1166417586.

[<img src="https://upload.wikimedia.org/wikipedia/commons/f/f3/CR_RNP_1.svg" width="48" alt="Primaria">](https://en.wikipedia.org/wiki/File:CR_RNP_1.svg) [<img src="https://upload.wikimedia.org/wikipedia/commons/1/13/CR_RNS_126.svg" width="48" alt="Secundaria">](https://en.wikipedia.org/wiki/File:CR_RNS_126.svg) [<img src="https://upload.wikimedia.org/wikipedia/commons/9/93/CR_RNT_606.svg" width="48" alt="Terceraria">](https://en.wikipedia.org/wiki/File:CR_RNT_606.svg)

Reply:
This sounds like a showstopper, right?  Don't we have a similar situation in a country in Europe?

Reply:
Yeah, I think so. We must be able to determine shield appearance from `network=*`, and three different route shields are all assigned `network=CR:national`. This would need to be fixed before we can support Costa Rican shields.

Reply:
I asked the [talk-cr](https://lists.openstreetmap.org/pipermail/talk-cr/) mailing list for help.